File: tests/fixtures/sample.cpp
int add(int a, int b) {
    return a + b;
}

int multiply(int x, int y) {
    int product = add(x, y);
    return product;
}

class Calculator {
public:
    int compute(int n) {
        return multiply(n, 2);
    }
};

File: tests/fixtures/sample.ts
function add(a: number, b: number): number {
  return a + b;
}

function multiply(x: number, y: number): number {
  const product = add(x, y);
  return product * 1;
}

class Calculator {
  compute(n: number): number {
    return multiply(n, 2);
  }
}
